locales/nl.js

```diff
@@ -0,0 +1,26 @@
+{
+	"DISBOARD: The Public Server List": "DISBOARD: De Openbare Server Lijst",
+	"Hi! I am a bot for DISBOARD (<https://disboard.org/>)": "Hi! Ik ben een bot voor DISBOARD (<https://disboard.org/>)",
+	"COMMAND LIST": "COMMANDO LIJST",
+	"This!": "Dit!",
+	"Bump this server": "Bump deze server",
+	"Get server page link": "Krijg server pagina link",
+	"Change Instant Invite to this channel. If [channel] is specified, create Instant Invite for that channel": "Verander Directe Uitnodiging naar deze kanaal. Als [channel] is gespecificeerd, maak Directe Uitnodiging voor dat kanaal",
+	"Admin only": "Alleen administratoren",
+	"Bump done": "Bump klaar",
+	"Check it on DISBOARD": "Bekijk het op DISBOARD",
+	"Please give me `Create Instant Invite` permission on this channel": "Alsjeblieft geef me de `Uitnodiging creëren` permissie voor deze kanaal",
+	"And then send this command: `%s`": "En gebruik dan deze commando: `%s`",
+	"Send this command to create invite: `%s`": "Stuur deze commando om de uitnodiging te updaten: `%s`",
+	"Invite link has been changed": "Uitnodiging link is veranderd",
```

Comment:
'Uitnodigingslink'. Link isn't _really_ a Dutch word but it's a commonplace loan word so it's fine.